package.json

```diff
@@ -39,11 +39,11 @@
     "sqlite3": "^3.0.8"
   },
   "peerDependencies": {
-    "knex": "> 0.8 < 0.96"
```

Comment:
This is actually excluding v2.0.

Reply:
yep thats why I changed it, you're reading the wrong side of the diff, what you have highlighted is the old par
<img width="1150" alt="Screen Shot 2022-05-12 at 10 00 51" src="https://user-images.githubusercontent.com/22284209/167983429-98dec415-a380-4528-b65b-728452e037d7.png">
t